exporter/jaeger/jaeger.go

```diff
@@ -107,13 +115,41 @@ func NewExporter(o Options) (*Exporter, error) {
 	return e, nil
 }
 
+// Tag defines a key-value pair
+// It is limited to the possible conversions to *jaeger.Tag by attributeToTag
+type Tag struct {
+	key   string
+	value interface{}
+}
+
+// BoolTag creates a new tag of type bool, exported as jaeger.TagType_BOOL
+func BoolTag(key string, value bool) Tag {
+	return Tag{key, value}
+}
+
+// StringTag creates a new tag of type string, exported as jaeger.TagType_STRING
+func StringTag(key string, value string) Tag {
+	return Tag{key, value}
+}
+
+// Int64Tag creates a new tag of type int64, exported as jaeger.TagType_LONG
+func Int64Tag(key string, value int64) Tag {
+	return Tag{key, value}
+}
+
+// Int32Tag creates a new tag of type int32, exported as jaeger.TagType_LONG
+func Int32Tag(key string, value int32) Tag {
```

Comment:
What's the reason for adding this? Seems like it is exported as TagType_LONG as well so why not just rely on the previous function?

Reply:
True, I added both to have a similar interface to other Tags, and support all 4 cases of `attributeToTag` (which handles both int32 and int64). I can remove it if you think it is not necessary?

Reply:
Yes, I think we should remove it if it's not required.